cmds/cmd_utils.py

```diff
@@ -197,6 +222,13 @@ async def grab_guild_info(message, args, client, **kwargs):
         'cache': 'keep',
         'execute': avatar_function
         },
+    'embed': {
+        'pretty_name': 'embed [text]',
+        'description': 'Creates an embed (requires mod)',
+        'permission': 'moderator',
+        'cache': 'regenerate',  # Make sure embed is credited to user
```

Comment:
I think this is comes down to an error with my poor documentation on commands, but what cache regeneration does is deletes and recreates the guilds configuration cache file from the database, by doing this you can process large amounts of message without needing to ever hit the database, lowering processing time  
It does not make sense to be regenerating the cache when a new embed is generated, as it does not modify any data in the database configuration cache that would warrant regenerating it